spdockit-insights/configure-data-collection.md

```diff
@@ -29,8 +29,8 @@ Use SPDocKit Insights to keep multiple farms under control and find problems mor
     - **Max Index Size** – by default we set this value to 200GB with a maximum of 1TB. Feel free to change it to suit your needs and hardware capabilities.
         - If the service determines that the index exceeds the maximum index size, it will stop the data collection. Note that the actual size of the index can vary because of index reorganization and can require up to 3 times the amount of disk space specified here.
 4. Confirm by clicking the OK button.
-    - Take note of the URL in the confirmation message. This is needed to connect to the SPDocKit Insights Agent if the SPDocKit Insights Application is installed on a separate server or workstation.
-    - If the SPDocKit Insights application is installed on the same server as the SPDocKit Insights Agent, SPDocKit Insights will start automatically.
+    - Take note of the URL in the confirmation message. This is needed to connect to the SPDocKit Insights Agent if the SPDocKit Insights Viewer is installed on a separate server or workstation. If that is the case copy the URL, start the Viewer on whatever server or workstation you installed it, and copy it into the required field. 
```

Comment:
If that is the case, copy the URL ~,~ and start the Viewer on whatever server of workstation you installed it on, and ~copy~ paste it into the required field.